### ISSUE TITLE
Sub-Menus

### ISSUE THREAD

**Issue by isaacplmann** [isaacplmann/ngx-contextmenu#3]
_From @AndreAlves10 on March 1, 2017 10:54_

Hello @isaacplmann,

Is it possible to create sub-menus? Just like the following:

![submenu](https://cloud.githubusercontent.com/assets/25454839/23456804/dc093294-fe6c-11e6-9a3a-914a5c0e74be.png)

I have an angular-cli project, in my Typescript file I have the following:

```
self.contextMenuService.show.next({
        actions: self.contextMenuItems,
        event: props.event,
        item: props.event.target
      });
```

And my HTML is like:
```
<context-menu>
    <template contextMenuItem (execute)="showMessage('Hi, ' + $event.item.name)">
        Say hi!
        <template contextMenuItem (execute)="showMessage('Hi, ' + $event.item.name)">
            Say hi!
        </template>
    </template>
    <template contextMenuItem divider="true"></template>
    <template contextMenuItem let-item (execute)="showMessage($event.item.name + ' said: ' + $event.item.otherProperty)">
        Bye, {{item?.name}}
    </template>
    <template contextMenuItem passive="true">
        Input something: <input type="text">
    </template>
</context-menu>
```

Thank you!

_Copied from original issue: isaacplmann/angular2-contextmenu#52_

**Comment by isaacplmann** [comment]
Not possible right now. That would be a great feature though. I'd love a pr.

**Comment by isaacplmann** [comment]
_From @AndreAlves10 on March 1, 2017 11:6_

Ok, thank you @isaacplmann!

**Comment by isaacplmann** [comment]
This is now implemented in version 1.2.0

**Comment by shomid** [comment]
> This is now implemented in version 1.2.0

how to implement ?
is this feature ?
